Makefile

```diff
@@ -34,3 +34,10 @@ run-migrations:
 	@kubectl -n gitlint-bot delete job/gitlint-bot-db-migrations --ignore-not-found
 	@kubectl -n gitlint-bot apply -f ./deploy/migrations-job.yaml
 	@kubectl -n gitlint-bot wait --for=condition=complete job/gitlint-bot-db-migrations --timeout=30s
+
+upload-sourcemaps:
+	@npm run build
+	@sentry-cli releases new ${SENTRY_RELEASE} --project ${SENTRY_PROJECT}
```

Comment:
In case you miss this: please remove the Sentry release flow, we don't need this at the moment: